From 86e517f09f29c7c45281c9ecc947fb1172f8f408 Mon Sep 17 00:00:00 2001
From: Nolen Johnson <johnsonnolen@gmail.com>
Date: Mon, 17 Sep 2018 23:08:46 +0000
Subject: [PATCH] Updater: Allow Cleartext HTTP communications

* As of 9.0, Android disallows ClearText traffic over
  HTTP by default. Re-Enable it for those of us using
  an updter instance that can't use SSL (i.e. behind
  a proxy, etc.).

Change-Id: I76f96e6424ce26922e9e196a0e1243b8494dc084
---
 AndroidManifest.xml | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/AndroidManifest.xml b/AndroidManifest.xml
index 0b0352f..0cfa6e8 100644
--- a/AndroidManifest.xml
+++ b/AndroidManifest.xml
@@ -16,7 +16,8 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
+        android:theme="@style/AppTheme"
+        android:usesCleartextTraffic="true" >
 
         <activity
             android:name=".UpdatesActivity"
-- 
2.17.1

